src/Infrastructure/LeanCode.ExternalIdentityProviders/Facebook/FacebookClient.cs

```diff
@@ -120,12 +124,18 @@ private FacebookUser ConvertResponse(JsonElement result)
 
         private string GenerateProof(string accessToken)
         {
-            logger.Debug("Signing Facebook request enabled - signing request");
-
-            var bytes = ParseKey(accessToken);
-            var hash = hmac.ComputeHash(bytes);
+            if (hmac is null)
+            {
+                logger.Error("Facebook's AppSecret is not configured, login with Facebook will fail");
```

Comment:
I'm not sure if the best place for this message is here or in the constructor, but both seem fine to me.

Reply:
I think here it will be easier to not miss the message.